src/actions/task-definitions.js

```diff
@@ -7,9 +7,7 @@ import {
   viewModeIsEditor,
   getCells,
   getCellBelowSelectedId,
-  getCellAboveSelectedId,
-  prettyDate,
-  formatDateString,
+  getCellAboveSelectedId, prettyDate, formatDateString,
```

Comment:
why this change?

Reply:
Oh I think that was a weird linting thing tbh

Reply:
i have been hearing such good things about "prettier". we should consider using that so that we never have to think/talk about boring linting stuff like evar. :-)